src/nupic/algorithms/Svm.cpp

```diff
@@ -163,7 +163,7 @@ namespace nupic {
 	
 	for (int i = 0; i < size(); ++i) {
 
-#if defined(NTA_PLATFORM_win32) && defined(NTA_COMPILER_MSVC)
+#if defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC)
```

Comment:
> just a nit, but for MSVC the ifdef WIN seems unnecessary? 

Could be, but with Visual Studio Community version in good progress, there's not guarantee that MSVC will continue being Windows only..